### ISSUE TITLE
Error with eager_load_one and eager_load_many when using "::" type conversion

### ISSUE THREAD

**Issue by aglushkov** [jhollinger/occams-record#11]
I was selecting fields like `SELECT (bulk_discount_data::json ->> 'percent')::float ...` and Occam tries to find `:json`, and `:float` variables
```
/occams-record-1.10.0/lib/occams-record/raw_query.rb:253:in `%': key{json} not found (KeyError)
/occams-record-1.10.0/lib/occams-record/raw_query.rb:253:in `escaped_sql'
/occams-record-1.10.0/lib/occams-record/raw_query.rb:111:in `run'
/occams-record-1.10.0/lib/occams-record/eager_loaders/ad_hoc_base.rb:57:in `run'
```

**Comment by aglushkov** [comment]
I know it can be fixed with changing `::` to `CAST` function, but anyway it looks like a bug.
I can add a PR with some fix if you don't mind

**Comment by jhollinger** [comment]
@aglushkov Good find! Yes, would be happy to accept a PR.

**Comment by jhollinger** [comment]
Released in v1.11.1
